src/main/java/com/juniorstart/juniorstart/model/User.java

```diff
@@ -52,4 +52,9 @@
     private AuthProvider provider;
 
     private String providerId;
+    
+    @OneToOne(fetch = FetchType.LAZY,
+            cascade =  CascadeType.ALL,
+            mappedBy = "user")
+    private JobOffer jobOffer;
```

Comment:
Hmm, co tutaj jest konkretnie podpinane? Jaką ofertę dany użytkownik wystawił? Jeśli tak, to trzeba pomyśleć nad listą ofert. Jeden użytkownik może chcieć ich wystawić więcej :P

Reply:
@OneToMany relacja